MeasureAuthoringTool/mat-drupal
# List view styles

## Description
Implement styles for content teasers and list views. Will be used for pages like the "News & Alerts" page shown in the mockups.

## To Test
1. Run pattern lab (`npm start` from the `html/themes/custom/emat` dir)
2. Visit the Teaser molecule and the List View organism to view those styled components, check that they appear the way they should according to the mockups.

### Teaser molecule
<img width="810" alt="Screen Shot 2020-02-17 at 3 50 45 PM" src="https://user-images.githubusercontent.com/1919263/74688491-50785480-519d-11ea-83e7-2e45633f1197.png">

### List View organism
<img width="805" alt="Screen Shot 2020-02-17 at 3 48 49 PM" src="https://user-images.githubusercontent.com/1919263/74688457-38a0d080-519d-11ea-99ed-823390afb7ec.png">
